plugins/gpio/GPIOPort.cpp

```diff
@@ -44,11 +44,11 @@ bool GPIOOutputPort::Init() {
 }
 
 string GPIOOutputPort::Description() const {
-  vector<uint8_t> pins = m_driver->PinList();
+  vector<uint16_t> pins = m_driver->PinList();
   std::ostringstream str;
   str << "Pins ";
 
-  vector<uint8_t>::const_iterator iter = pins.begin();
+  vector<uint16_t>::const_iterator iter = pins.begin();
```

Comment:
I've had a busy couple weeks so I haven't checked in on this - I'll try to look into this this week and see if I can do that.